README.md

```diff
@@ -134,8 +136,24 @@ Learn more about ONNX
 - [ONNX website](http://onnx.ai/)
 - [ONNX on GitHub](https://github.com/onnx/onnx)
 
-### Operators supported
-The [list](./docs/operators.md) of ONNX operators supported by each of the 3 available builtin backends (cpu, wasm, and webgl).
+### Compatibility
+#### Desktop Platforms
+| OS/Browser | Chrome | Edge | FireFox | Safari | Opera | Electron | Node.js |
+|:----------:|:------:|:----:|:-------:|:------:|:-----:|:-----:|:-----:|
+| Windows 10 | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | - | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| macOS | :heavy_check_mark: | - | :heavy_check_mark: | Coming soon | :heavy_check_mark: | Coming soon | Coming soon |
```

Comment:
should take a look at Node.js on macOS. 

I think this should already supported